src/app/tab2/tab2.page.html

```diff
@@ -20,6 +20,8 @@
       <ion-select-option value="block">Wood Block</ion-select-option>
     </ion-select>
   </ion-item>
+  <ion-button size="small" fill="outline" (click)="enableFawhnification()">Fawhnify</ion-button>
```

Comment:
would probably like to see this button pair on the main page instead of on the already pretty cluttered timer page